experimenter/experimenter/nimbus-ui/src/components/Summary/index.test.tsx

```diff
@@ -74,6 +75,20 @@ describe("Summary", () => {
       screen.queryByTestId("summary-page-signoff-not-launched"),
     ).toBeInTheDocument();
   });
+  it("updates signoff checklist state correctly", () => {
+    render(<Subject />);
+    const legalSignoffCheckbox = screen.getByTestId("is-legalsignoff-checkbox");
+    const qaSignoffCheckbox = screen.getByTestId("is-qasignoff-checkbox");
+    const vpSignoffCheckbox = screen.getByTestId("is-vpsignoff-checkbox");
+
+    userEvent.click(legalSignoffCheckbox);
+    userEvent.click(qaSignoffCheckbox);
+    userEvent.click(vpSignoffCheckbox);
+
+    expect(legalSignoffCheckbox).toBeChecked();
+    expect(qaSignoffCheckbox).toBeChecked();
+    expect(vpSignoffCheckbox).toBeChecked();
```

Comment:
I would prefer the test to do:

- click legal signoff
- check legal signoff is checked
- click qa signoff
- check qa signoff is checked
- click vp signoff
- check vp signoff is checked